src/app/Instance/ErrorHandling/ErrorHandlingEdit.tsx

```diff
@@ -71,55 +74,71 @@ export const ErrorHandlingEdit = ({
   }, [isSubmitted]);
 
   return (
-    <Form
-      className="error-handling-edit-form"
-      autoComplete="off"
-      onSubmit={onSubmit}
+    <PageSection
+      variant={PageSectionVariants.light}
+      isFilled
+      padding={{ default: "noPadding" }}
+      className={"pf-u-h-100"}
     >
-      {apiError && (
-        <FormAlert>
-          <Alert
-            className="error-handling-edit__alert"
-            ouiaId="error-schema"
-            variant="danger"
-            title={apiError}
-            aria-live="polite"
-            isInline
-          />
-        </FormAlert>
-      )}
-      <ErrorHandlingCreate
-        schemaId={errorHandlingMethod}
-        getSchema={getSchema}
-        registerValidation={registerValidateParameters}
-        onChange={onErrorHandlingParametersChange}
-        parameters={errorHandlingParameters}
-      />
-      <PageSection
-        stickyOnBreakpoint={{ default: "bottom" }}
-        padding={{ default: "noPadding" }}
-        style={{ boxShadow: "none" }}
+      <StickyActionsLayout
+        actions={
```

Comment:
Adding the new layout component and passing if the EH form and form actions as props,